src/base/cvar.ml

```diff
@@ -68,31 +68,30 @@ let to_constant_and_terms ~equal ~add ~mul ~zero ~one =
     let c = if equal c zero then None else Some c in
     (c, ts)
 
-module Make (Field : Snarky_intf.Field.Extended) :
-  Intf with type field := Field.t = struct
+module Make (Field : Snarky_intf.Field.S) : Intf with type field := Field.t =
```

Comment:
It seems `Snarky_intf.Field.Extended` is just `Snarky_intf.Field.S` with some helpers extended. I guess we're relaxing the assumption on the module here?